proto/p4/v1/p4runtime.proto

```diff
@@ -240,6 +240,9 @@ message FieldMatch {
     Ternary ternary = 3;
     LPM lpm = 4;
     Range range = 6;
+    // Architecture-specific match value; it corresponds to the other_match_type
+    // in the P4Info MatchField message.
+    google.protobuf.Any other = 7;
```

Comment:
Might be a good idea to make this field ID 8 since valid used to be 7.

Reply:
Or 100 to give space for other intrinsic types.

Reply:
done